src/widget/footer/footer.jsx

```diff
@@ -0,0 +1,49 @@
+import './footer.scss';
+import ExpressLogo from '../../shared/ui/icons/express-logo';
+import Logo from '../../shared/ui/icons/logo';
+
+function Footer() {
+	return (
+		<footer className="footer">
+			<div className="footer__container">
+				<div className="footer__main">
+					<div className="footer__logo">
+						<Logo width="100px" height="100px" />
+					</div>
+					<div className="footer__column">
+						<ExpressLogo width="158px" height="70px" />
+						<div className="footer__main-contacts">
+							<div className="footer__phone-block">
+								<span className="footer__phone-number">8 880 111 2222</span>
+								<div>Круглосуточно</div>
+							</div>
+							<div className="footer__contacts">Контакты</div>
+						</div>
+						<div className="footer__documents">
+							<a href="/" className="footer__link">
+								Оферта
+							</a>
+							<a href="/" className="footer__link">
+								Политика конфидициальности
```

Comment:
```suggestion
								Политика конфиденциальности
```
стоит еще раз смотреть ПР сразу как открыл его, часто находишь много ошибок, сделанных по невнимательности